### ISSUE TITLE
Unable to create admin account

### ISSUE THREAD

**Issue by 360matt** [SPLRGE/traefik#6]
Issue:
- When I attempt to register with my credentials and apply, the form is refreshed, but nothing happened.

Informations:
- the CRSF token is present in the DOM, but seems not be goot (server log: ``Invalid or expired CSRF token``)

**Comment by vic256** [comment]
Fixed, upgrade to v0.0.0-beta.4 version. 
Maybe another we'll have to find another fix in the future.

**Comment by ChocolatMilka** [comment]
Not fixed again,
I just ``docker compose pull``

**Comment by ChocolatMilka** [comment]
With same error message ``Invalid or expired CSRF token``

**Comment by vic256** [comment]
Yup, it's a mistake. I disabled the CSRF protection on setup but not on the others pages as it could lead to security issues. I have to find a better option. 
As the dashboard should not be exposed, maybe this CSRF protection could be deactivated (@dev-cetus  ?).
